navigator/backend/src/main/scala/com/digitalasset/navigator/store/platform/PlatformStore.scala

```diff
@@ -159,21 +159,34 @@ class PlatformStore(
   }
 
   def connected(state: StateConnected): Receive = {
-    case UpdateParties =>
+    case UpdatePartiesAndUsers =>
       state.ledgerClient.partyManagementClient
         .listKnownParties()
         .map(UpdatedParties(_))
         .pipeTo(self)
+
+      state.ledgerClient.userManagementClient
+        .listUsers() // TODO what token should we pass?
```

Comment:
The ledger client API allows you to pass a token when creating the ledger client as well as on each request. If you don’t specify one per request we use the one used when creating the client (if any). Given that Navigator only accepts one token, we just set this on startup in https://github.com/digital-asset/daml/blob/f057765cd27ecefcdf903ab92a8b340357cdb5c6/navigator/backend/src/main/scala/com/digitalasset/navigator/store/platform/PlatformStore.scala#L317 and you shouldn’t have to worry about tokens here.

Reply:
Cool, replacing TODO with this explanation -- thanks!